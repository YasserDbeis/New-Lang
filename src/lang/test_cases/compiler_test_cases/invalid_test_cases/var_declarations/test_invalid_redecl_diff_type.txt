func main() {
    int x;
    dec x;
}