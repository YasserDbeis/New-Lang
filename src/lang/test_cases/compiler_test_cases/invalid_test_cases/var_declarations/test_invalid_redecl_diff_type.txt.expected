RUNTIME ERROR
Error occurred in Execution
Variable x has already been declared in this scope
Variable has already been declared
-1